packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -248,6 +250,19 @@ class Scrollable extends StatefulWidget {
   /// Determined by the [axisDirection].
   Axis get axis => axisDirectionToAxis(axisDirection);
 
+  /// {@template flutter.widgets.scrollable.autoScrollbar}
+  /// Whether a [Scrollbar] should automatically be applied to this Scrollable.
```

Comment:
This property only takes affect when a controller is also provided, though, right?